src/containers/TrackList.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux';
```

Comment:
This is gorgeous. Just the wiring up of the component, exactly as we discussed!

Reply:
Yay!